Applications/spire/include/spire/charting/chart_view.hpp

```diff
@@ -50,6 +50,24 @@ namespace Spire {
       //! Removes the crosshair from the chart.
       void reset_crosshair();
 
+      //! Passes mouse click events to this view.
+      /*
+        \param event The click event.
+      */
+      void mouse_click(QMouseEvent* event);
```

Comment:
The public API can not change. The ChartView doesn't know anything about the mouse, otherwise it would handle its own mouse events.

Reply:
So the charting window is managing all of this?

Reply:
Yep exactly.

Reply:
But then won't ChartView and ChartWindow have duplicate members for the region?